content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: An Alternative Istiod Deployment Model
+subtitle: Clear Separation between Mesh Operator and Mesh Admin
+description: A new deployment model for Istio.
+publishdate: 2020-08-26
+attribution: "Lin Sun (IBM), Iris Ding (IBM)"
+keywords: [istiod,deployment model,install,deploy,'1.7']
+---
+
+## Overview
+
+From experience working with various service mesh users and vendors, we believe there are 3 key personas for a typical service mesh:
+
+* Mesh Operator, who manages the service mesh control plane installation and upgrade.
+
+* Mesh Admin, often referred as Platform Owner, who owns the service mesh platform and defines the overall strategy and implementation for service owners to adopt service mesh.
+
+* Mesh User, often referred as Service Owner, who owns one or more services in the mesh.
+
+Istio currently requires the control plane to run in one of the clusters in the mesh, leading to a lack of separation between the mesh operator and the mesh admin. We're introducing an alternative deployment model for Istiod which enables mesh operators to install and manage the mesh control plane on separate clusters. This new deployment model allows a clear separation between mesh operators and mesh admins. With this new model, Istio mesh operators can run Istio control plane for mesh admins while mesh admins can still control the configuration of the control plane without worrying about installing or managing the Istio control plane. This model is transparent to mesh users.
+
+## External Control Plane Deployment model
+
+After installing Istio using the [default installation profile](/docs/setup/install/istioctl/#install-istio-using-the-default-profile), you will have an Istiod control plane installed in a single cluster like the diagram below:
```

Comment:
I maintain that it's not important that it's istiod, it's just that it's a control plane.  That's my opinion however, please feel free to ignore.

Reply:
Yes,  you are right.  We had Istiod here because that is what the diagram had so easier for user to read + match what is in the diagram.